### PR TITLE
Use glob to check if SAVE_PATH exists;Imporve Saliency map value;Use …

### PR DESCRIPTION
**Pull request by WangXin93**
1. 
Use glob to check if SAVE_PATH exists, beacause the squeezenet checkpoint has three files actually.
[Reference](https://github.com/cs231n/cs231n.github.io/issues/165)
2.
`loss = correct_scores # not tf.square(1-correct_scores)`
Experiment shows it can have clearer saliency map.
3.
`dX = learning_rate * g / tf.norm(g)`
instead of **tf.reduce_sum** can generate a fooling image with fewer noise. It is clearer while inputing a **quail** image.

### CODE REVIEW

**Comment by Shiaoming** [comment]
I checked these issues, and it proves you are right, thanks for your contribution! I will add the glob to check if SAVE_PATH exists right now.
